build.rs

```diff
@@ -357,10 +357,19 @@ fn blob_backends(conf: &Config, build_dir: &std::path::PathBuf) -> (Vec<String>,
     if conf.use_lib {
         let afpath  = match env::var("AF_PATH") {
             Ok(af_path) => PathBuf::from(&af_path),
-            Err(_)      => panic!("Error use_lib is defined, but AF_PATH is not defined"),
+            Err(_)      => {
+                println!("WARNING! USE_LIB IS DEFINED, BUT AF_PATH IS NOT DEFINED, TRYING TO FIND LIBRARIES FROM KNOWN DEFAULT LOCATIONS");
+                PathBuf::from("/usr/")
```

Comment:
If there is no AF_PATH, it will look for libraries in some predefined paths for platforms. If there is AF_PATH, it will also be included into the search paths.